### ISSUE TITLE
DRC: Als zaakbehandelaar wil ik een uitgaand document opstellen met een geadresseerde, voordat ik de verzenddatum weet

### ISSUE THREAD

**Issue by MNIJ** [VNG-Realisatie/gemma-zaken#2188]
... zodat ik de gegevens van de geadresseerde slechts eenmalig hoef vast te leggen.

In het object `Verzending` in de DRC kan nu worden vastgelegd van wie een document ontvangen is of aan wie het is verzonden. Momenteel is het verplicht om bij het aanmaken van een Verzending direct een `verzenddatum` mee te geven als `aardRelatie = geadresseerde` en het dus gaat om een uitgaand document.

Vaak wordt een document echter eerst opgesteld, dan nog tegengelezen en/of geaccordeerd en pas daarna daadwerkelijk verzonden. Hier zijn mogelijk meerdere personen bij betrokken en dit proces kan enige tijd duren. Het is dus bij het opstellen al wel bekend aan wie het document verzonden moet worden, maar nog niet exact wanneer. Mogelijk wordt het uiteindelijk zelfs helemaal niet verzonden.

Om het mogelijk te maken de geadresseerde al direct vast te leggen, waardoor deze gegevens slechts enkelvoudig vastgelegd hoeven te worden en bijvoorbeeld ook gebruikt kunnen worden in sjablonen voor automatische documentcreatie, is het wenselijk dat het bij uitgaande documenten niet verplicht is om de verzenddatum op te geven.

**Comment by michielverhoef** [comment]
Verzenddatum is niet verplicht. Ook in de huidige versie kun je een uitgaande verzending (aardrelatie = geadresseerde) aanmaken zonder verzenddatum.  Onderstaande heb ik met postman geprobeerd en dat gaat zonder problemen.

POST https://documenten-api.vng.cloud/api/v1/verzendingen
Authorization Bearer token uit "ZGW API TESTS referentiecomponenten"

requestbody: 
{
    "betrokkene": "http://example.com",
    "informatieobject": "https://documenten-api.vng.cloud/api/v1/enkelvoudiginformatieobjecten/3623d1c9-28a0-4141-baf6-7ce4dcdb77af",
    "aardRelatie": "geadresseerde",
    "toelichting": "DOCUMENT-2019-0000000001 Zonder verzenddatum",
    "contactPersoon": "http://example.com",
    "contactpersoonnaam": "Jansen",
    "binnenlandsCorrespondentieadres": 
    {
        "huisnummer": 12,
        "naamOpenbareRuimte": "Nassaulaan",
        "postcode": "2514JS",
        "woonplaatsnaam": "Den Haag"

    }
}

response: 
{
    "url": "https://documenten-api.vng.cloud/api/v1/verzendingen/bdaae5cc-693a-4d50-9192-f1b046d8cba0",
    "betrokkene": "http://example.com",
    "informatieobject": "https://documenten-api.vng.cloud/api/v1/enkelvoudiginformatieobjecten/3623d1c9-28a0-4141-baf6-7ce4dcdb77af",
    "aardRelatie": "geadresseerde",
    "toelichting": "DOCUMENT-2019-0000000001 Zonder verzenddatum",
    "ontvangstdatum": null,
    "verzenddatum": null,
    "contactPersoon": "http://example.com",
    "contactpersoonnaam": "Jansen",
    "binnenlandsCorrespondentieadres": {
        "huisletter": "",
        "huisnummer": 12,
        "huisnummerToevoeging": "",
        "naamOpenbareRuimte": "Nassaulaan",
        "postcode": "2514JS",
        "woonplaatsnaam": "Den Haag"
    },
    "buitenlandsCorrespondentieadres": {
        "adresBuitenland1": "",
        "adresBuitenland2": "",
        "adresBuitenland3": "",
        "landPostadres": ""
    },
    "correspondentiePostadres": {
        "postBusOfAntwoordnummer": null,
        "postadresPostcode": "",
        "postadresType": "",
        "woonplaatsnaam": ""
    },
    "faxnummer": null,
    "emailadres": null,
    "mijnOverheid": false,
    "telefoonnummer": null
}


POST https://documenten-api.vng.cloud/api/v1/verzendingen
Authorization Bearer token uit "ZGW API TESTS referentiecomponenten"

requestbody:
{
    "betrokkene": "http://example.com",
    "informatieobject": "https://documenten-api.vng.cloud/api/v1/enkelvoudiginformatieobjecten/3623d1c9-28a0-4141-baf6-7ce4dcdb77af",
    "aardRelatie": "geadresseerde",
    "toelichting": "DOCUMENT-2019-0000000001 OPENBAAR",
    "verzenddatum": "2019-08-24",
    "contactPersoon": "http://example.com",
    "contactpersoonnaam": "Jansen",
    "binnenlandsCorrespondentieadres": 
    {
        "huisnummer": 12,
        "naamOpenbareRuimte": "Nassaulaan",
        "postcode": "2514JS",
        "woonplaatsnaam": "Den Haag"

    }
}

response: 
{
    "url": "https://documenten-api.vng.cloud/api/v1/verzendingen/8d6b3420-80d7-4be7-b955-025c7164011d",
    "betrokkene": "http://example.com",
    "informatieobject": "https://documenten-api.vng.cloud/api/v1/enkelvoudiginformatieobjecten/3623d1c9-28a0-4141-baf6-7ce4dcdb77af",
    "aardRelatie": "geadresseerde",
    "toelichting": "DOCUMENT-2019-0000000001 OPENBAAR",
    "ontvangstdatum": null,
    "verzenddatum": "2019-08-24",
    "contactPersoon": "http://example.com",
    "contactpersoonnaam": "Jansen",
    "binnenlandsCorrespondentieadres": {
        "huisletter": "",
        "huisnummer": 12,
        "huisnummerToevoeging": "",
        "naamOpenbareRuimte": "Nassaulaan",
        "postcode": "2514JS",
        "woonplaatsnaam": "Den Haag"
    },
    "buitenlandsCorrespondentieadres": {
        "adresBuitenland1": "",
        "adresBuitenland2": "",
        "adresBuitenland3": "",
        "landPostadres": ""
    },
    "correspondentiePostadres": {
        "postBusOfAntwoordnummer": null,
        "postadresPostcode": "",
        "postadresType": "",
        "woonplaatsnaam": ""
    },
    "faxnummer": null,
    "emailadres": null,
    "mijnOverheid": false,
    "telefoonnummer": null
}

**Comment by MNIJ** [comment]
@michielverhoef 
Dit moet dan nog wel worden aangepast in de documentatie. Daar staat nu in de definitie van het veld `verzenddatum` van een verzending deze tekst:

> De datum waarop het INFORMATIEOBJECT verzonden is, zoals deze op het INFORMATIEOBJECT vermeld is. Dit geldt voor zowel inkomende als uitgaande INFORMATIEOBJECTen. Eenzelfde informatieobject kan niet tegelijk inkomend en uitgaand zijn. Ontvangst en verzending is voorbehouden aan documenten die van of naar andere personen ontvangen of verzonden zijn waarbij die personen niet deel uit maken van de behandeling van de zaak waarin het document een rol speelt. Vervangt het gelijknamige attribuut uit Informatieobject. **Verplicht gevuld wanneer aardRelatie de waarde 'geadresseerde' heeft**.

**Comment by HenriKorver** [comment]
Ja, ik zie het. Ik zal de foute tekst corrigeren in de volgende release.

**Comment by MNIJ** [comment]
> [@MNIJ](https://github.com/MNIJ) Foute tekst is aangepast, zie o.a. hier
> 
> https://vng-realisatie.github.io/gemma-zaken/standaard/documenten/redoc-1.6.0#tag/verzendingen/operation/verzending_create
> 
> Is het zo naar wens?

Ja, dit ziet er prima uit!
